src/Components/Aspire.Azure.AI.Inference/ChatCompletionsClientSettings.cs

```diff
@@ -4,9 +4,8 @@
 using System.Data.Common;
 using Aspire.Azure.Common;
 using Azure.Core;
-using Microsoft.Identity.Client.Platforms.Features.DesktopOs.Kerberos;
 
-namespace Microsoft.Extensions.Hosting;
+namespace Aspire.Azure.AI.Inference;
```

Comment:
Is this a breaking change?

Reply:
Yes, but it is in a "preview" package. So we should be OK. Can just doc it.